samples/csharp_dotnetcore/48.customQABot-all-features/wwwroot/default.htm

```diff
@@ -0,0 +1,417 @@
+﻿<!DOCTYPE html>
```

Comment:
rename file to .html extension (like samples/csharp_dotnetcore/02.echo-bot/wwwroot/default.html)

Reply:
[CoreBot ](https://github.com/microsoft/BotBuilder-Samples/tree/main/samples/csharp_dotnetcore/13.core-bot/wwwroot) has **default.htm**. Shall we change it here?

Reply:
> [CoreBot ](https://github.com/microsoft/BotBuilder-Samples/tree/main/samples/csharp_dotnetcore/13.core-bot/wwwroot) has **default.htm**. Shall we change it here?

yes please.